src/css/imports/title.less

```diff
@@ -32,7 +31,7 @@
     margin-top: -.5em;
 }
 
-.jw-flag-small-player {
+.jw-flag-small-player:not(.jw-flag-audio-player) {
```

Comment:
Please remove the not. It makes this rule more specific which can introduce regressions in other areas